internal/otelarrow/admission/boundedqueue_test.go

```diff
@@ -5,209 +5,330 @@ package admission
 
 import (
 	"context"
+	"fmt"
```

Comment:
Note that this file has been rewritten from scratch because of the FIFO->LIFO change and the fallibility change. This file brings us to 100% test coverage.